Meadowlark-js/services/meadowlark-fastify/.env.example

```diff
@@ -50,7 +51,7 @@ FASTIFY_PORT=3000
 # This "stage" name will appear in the API routes
 MEADOWLARK_STAGE=local
 
-TOKEN_URL: http://localhost:3000/local/api/oauth/token
```

Comment:
⚠ This is why I was getting failures: that `api/` in this example URL is not correct. So all of my requests to `api/oauth/token` were being handled as regular resource requests, rather than as oauth requests. I didn't get a 404 on there because the authorization middleware first rejected with 401.

I don't know when we lost the `api/`. We may need to restore that everywhere. It isn't strictly necessary of course. But quite customary and might be confusing to API clients.